src/Tickets/Commerce/Gateways/Stripe/Signup.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Gateways\Stripe;
+
+use TEC\Tickets\Commerce\Gateways\Contracts\Abstract_Signup;
+use TEC\Tickets\Commerce\Gateways\Stripe\REST\On_Boarding_Endpoint;
+
+class Signup extends Abstract_Signup {
+
+	/**
+	 * @inheritDoc
+	 */
+	public static $signup_data_meta_key = 'tec_tc_stripe_signup_data';
+
+	/**
+	 * The return path the user will be redirected to after signing up
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $signup_return_path = '/edit.php?post_type=tribe_events&page=tribe-common&tab=payments';
```

Comment:
```suggestion
	public $signup_return_path = '/admin.php?page=tribe-common&tab=payments';
```
This should work fine without `tribe_events` which is dependent on TEC installation.